Caboodle/Browser/Browser.shared.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    /// <summary>
+    /// Shared code between Browser implementations
+    /// Contains static methods and shared members
+    /// </summary>
+    public partial class Browser
+    {
+        /// <summary>
+        /// </summary>
+        /// TODO: naming discussion
+        /// Simple Launching
+        /// Android     - intent.StartActivity(..., Uri)
+        /// iOS         - SharedApplcation.OpenUrl(NSUrl)
+        /// UWP         - Launcher.LaunchUriAsync(Uri)
+        /// suggestion for naming
+        /// moljac
+        /// UseUriLauncher / UseLanucher
+        /// UseSystemBrowser
+        public static bool AlwaysUseExternal
```

Comment:
This makes the `OpenAsync` calls a bit ambiguous... Presumably this would override whatever was passed into that method.  I think we should remove this static override and force an explicit choice within `OpenAsync` if the user wants to open in external.